docs/docs/system-administration/architecture.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Architecture
```

Comment:
I'm not sure that I totally agree that this belongs in the 'System Administration' section... but I don't know that I have a better suggestion without creating a totally new section in the nav.

Reply:
Yeah it has never really had anywhere decent to live. I think this is the least worst option